SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -150,7 +150,7 @@ To resolve this you need to mark the `CurrentHealth` property for replication, j
 
 Notice that the workflow you just used mirrors that of native Unreal.
 
-Because you’ve changed code in a function you now need to build your project, generate schema and a new snapshot. To do this:
+Because you have changed code in a function you now need to rebuild your project. Additionally, because you have enabled replication for a variable you need to generate schema and a new snapshot. To do this:
```

Comment:
```suggestion
Because you have changed code in a function, you now need to rebuild your project. Additionally, because you have enabled replication for a variable, you need to generate schema and a new snapshot. To do this:
```

Reply:
added